ghcide-bench/ghcide-bench.cabal

```diff
@@ -108,7 +108,7 @@ test-suite test
         implicit-hie:gen-hie
     main-is: Main.hs
     hs-source-dirs: test
-    ghc-options: -Wunused-packages
+    ghc-options: -Wunused-packages -Werror=unused-packages
```

Comment:
I think we probably don't want the `Werror`? Or can we do this only in CI? any kind of `Werror` is a huge pain during development IMO

Reply:
In fact, don't we already have a `pedantic` cabal flag that we set to enable `Werror` during CI?